pkg/utils/utils.go

```diff
@@ -193,9 +195,34 @@ func GetS3PETMFromParamsMap(params map[string]interface{}, logger logrus.FieldLo
 		return nil, errors.New("Missing bucket param, cannot initialize S3 PETM")
 	}
 
-	sess := session.Must(session.NewSession(&aws.Config{
-		Region: aws.String(region),
-	}))
+	useMinio := true
+	s3Url, ok := params["s3Url"].(string)
+	if !ok {
+		useMinio = false
+	}
+	pathStyle, ok := params["s3ForcePathStyle"].(string)
+	if !ok {
+		useMinio = false
+	}
+	var s3ForcePathStyl bool
+	if pathStyle == "true" {
+		s3ForcePathStyl = true
+	} else {
+		s3ForcePathStyl = false
+	}
+
+	var sess *session.Session
+	if useMinio {
+		sess = session.Must(session.NewSession(&aws.Config{
```

Comment:
Let's change this so we make an aws.Config object and then set the Endpoint/S3ForcePathStyle based on whether they exist or not.  Then you don't need the "useMinio" or "useS3Path" variable.